pootle/static/js/editor/index.js

```diff
@@ -12,15 +12,18 @@ import React from 'react';
 import ReactRenderer from 'utils/ReactRenderer';
 
 import Editor from './containers/Editor';
+import UnitSource from './components/UnitSource';
 import { hasCRLF, normalize, denormalize } from './utils/normalizer';
 
 
 const ReactEditor = {
 
   init(props) {
     this.node = document.querySelector('.js-mount-editor');
+    this.sourceNode = document.querySelector('.js-mount-editor-original-src');
```

Comment:
Good to know: now we have `q` and `qAll` in `utils/dom` as aliases over `document.querySelector` and `document.querySelectorAll`. The latter will provide a native `Array` instead of a `NodeList` (which is Array-like but doesn't support some iterating methods such as `forEach`).